README.md

```diff
@@ -626,6 +626,7 @@ For the `spring` generator, the additional configuration options are:
 | `useClassLevelBeanValidation` | Adds @Validated annotation to API interfaces (Default: false) |
 | `useLombokAnnotations` | Use Lombok annotations to generate properties accessors and `hashCode`/`equals` methods (Default: false) |
 | `addServletRequest` | Adds ServletRequest objects to API method definitions (Default: false) |
+| `addBindingResult` | Adds BindingResult to Api method definitions' request bodies, if UseBeanValidation configured, for this to be effective use configuration UseBeanValidation (Default: false)|
```

Comment:
Does addBindingResult also enable useBeanValidation? The description here is a bit awkward.